docs/data_management/res/cheaha_project_directory.csv

```diff
@@ -0,0 +1,10 @@
+Permissions/Responsibilities,Research Computing,Members,Manager,PI
```

Comment:
As discussed, let's flip the table upside-down so the most-full rows are at the top, least full at the bottom. Move RC responsibilities to the right-most column.